docs/program/sdks/_index.md

```diff
@@ -20,11 +21,12 @@ Viam offers software development kits (SDKs) in popular languages which
 
 Use the SDK of your preferred language to write code to control your robots.
 
-Viam currently offers SDKs for the following three languages:
+Viam currently offers SDKs for the following languages:
 
 - [Python SDK](https://python.viam.dev/)
 - [Go SDK](https://pkg.go.dev/go.viam.com/rdk)
 - [TypeScript SDK](https://ts.viam.dev/)
+- [C++ SDK](https://cpp.viam.dev/)
```

Comment:
opt: should we note "alpha" here? Perhaps cumbersome to remember to change later, but without such notice, these SDKs seem like equal citizens in this list.

Reply:
```suggestion
- [C++ SDK (alpha)](https://cpp.viam.dev/)
```